src/material/list/selection-list.ts

```diff
@@ -72,8 +72,14 @@ export class MatSelectionListChange {
   constructor(
     /** Reference to the selection list that emitted the event. */
     public source: MatSelectionList,
-    /** Reference to the option that has been changed. */
-    public option: MatListOption) {}
+    /**
+     * Reference to the option that has been changed.
+     * @deprecated Use `options` instead, because some events may change more than one option.
+     * @breaking-change 12.0.0
+     */
+    public option: MatListOption,
+    /** Reference to the options that have been changed. */
+    public options: MatListOption[]) {}
```

Comment:
If we wanted to avoid this future breaking change, we could also emit one event per selected option like we do in `mat-select`. I decided not to do it, because the `mat-select` behavior feels broken.